src/tequila/trotter_err/main_trot.py

```diff
@@ -128,8 +128,9 @@ def EstTrotErrParal(ListFrags,nqubs,SymDict=None):
 
     #mp.set_start_method('spawn')
     # Define the function that computes alpha_2 for a pair of fragments
-    pool = mp.Pool(mp.cpu_count())
-    #with mp.Pool(mp.cpu_count()) as pool:
+    if pool==None:
```

Comment:
great!
One tiny thing: it needs to be "if pool is None" so "is" instead if "==".
In practice it behaves the same, but the syntax checker in the testing suite will not let this pass :-)